docs/guide/api-plugin.md

```diff
@@ -25,6 +25,48 @@ If your plugin is only going to work for a particular framework, its name should
 - `vite-plugin-react-` prefix for React Plugins
 - `vite-plugin-svelte-` prefix for Svelte Plugins
 
+## Plugins config
+
+Users will add plugins to the project `devDependencies` and configure them using the `plugins` array option.
+
+```js
+// vite.config.js
+import vitePlugin from 'vite-plugin-feature'
+import rollupPlugin from 'rollup-plugin-feature'
+
+export default {
+  plugins: [ vitePlugin(), rollupPlugin() ]
+}
+```
+
+Falsy plugins will be ignored, which can be used to easily activate or deactivate plugins. 
+
+`plugins` also accept presets including several plugins as a single element. This is useful for complex features (like framework integration) that are implemented using several plugins. The array will be flattened internally.
+
+```js
+// framework-plugin
+import frameworkRefresh from "vite-plugin-framework-refresh";
```

Comment:
**suggestion (if-minor):** You should keep consistent quote style
Above in line 34 you use single quotes